sql/plan/window.go

```diff
@@ -143,7 +143,7 @@ func (i *windowIter) Next() (sql.Row, error) {
 				return nil, err
 			}
 		case sql.Aggregation:
-			row[j], err = expr.Eval(i.ctx, i.buffers[j])
+			row[j], err = i.buffers[j][0].(sql.AggregationBuffer).Eval(i.ctx)
```

Comment:
This function could use a TODO, it needs to be given the same treatment as the non-window aggregates